docs/installation/getting-started/quick-start.md

```diff
@@ -4,318 +4,770 @@
+				'|',
+				'textPartLanguage',
+				'|',
+				'sourceEditing'
+			],
+			shouldNotGroupWhenFull: true
+		},
+	image: {
+			styles: [
+				'alignCenter',
+				'alignLeft',
+				'alignRight'
+			],
+			resizeOptions: [
+				{
+					name: 'resizeImage:original',
+					label: 'Original',
+					value: null
+				},
+				{
+					name: 'resizeImage:50',
+					label: '50%',
+					value: '50'
+				},
+				{
+					name: 'resizeImage:75',
+					label: '75%',
+					value: '75'
+				}
+			],
+			toolbar: [
+				'imageTextAlternative', 'toggleImageCaption', '|',
+				'imageStyle:inline', 'imageStyle:wrapText', 'imageStyle:breakText', 'imageStyle:side', '|',
+				'resizeImage'
+			],
+			insert: {
+				integrations: [
+					'insertImageViaUrl'
+				]
+			}
+		},
+		list: {
+			properties: {
+				styles: true,
+				startIndex: true,
+				reversed: true
+			}
+		},
+		link: {
+			decorators: {
+				addTargetToExternalLinks: true,
+				defaultProtocol: 'https://',
+				toggleDownloadable: {
+					mode: 'manual',
+					label: 'Downloadable',
+					attributes: {
+						download: 'file'
+					}
+				}
+			}
+		},
+		mention: {
+			feeds: [
+				{
+					marker: '@',
+					feed: [
+						'@apple', '@bears', '@brownie', '@cake', '@cake', '@candy', '@canes', '@chocolate', '@cookie', '@cotton', '@cream',
+						'@cupcake', '@danish', '@donut', '@dragée', '@fruitcake', '@gingerbread', '@gummi', '@ice', '@jelly-o',
+						'@liquorice', '@macaroon', '@marzipan', '@oat', '@pie', '@plum', '@pudding', '@sesame', '@snaps', '@soufflé',
+						'@sugar', '@sweet', '@topping', '@wafer'
+					],
+					minimumCharacters: 1
+				}
+			]
+		},
+		placeholder: 'Type or paste your content here!',
+		table: {
+			contentToolbar: [
+				'tableColumn', 'tableRow', 'mergeTableCells', 'tableProperties', 'tableCellProperties', 'toggleTableCaption'
+			]
+		},
 	} )
 	.catch( error => {
-		console.error( error.stack );
+		console.log( error );
 	} );
 ```
 
-You can now run webpack to build the application. To do that, call the `webpack` executable:
+### Sample implementation
 
-```bash
-./node_modules/.bin/webpack --mode development
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>CKEditor 5 – Full-featured editor</title>
+	<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/super-build/ckeditor.js"></script>
+</head>
+<body>
+	<h1>Full-featured editor</h1>
+	<div id="editor">
+		<p>This is some sample content.</p>
+	</div>
+	<script>
+		CKEDITOR.ClassicEditor
+			.create( document.querySelector( '#editor' ), {
+				removePlugins: [ 'Comments, RealTimeCollaborativeComments, TrackChanges, RealTimeCollaborativeTrackChanges, RevisionHistory, RealTimeCollaborativeRevisionHistory, RealTimeCollaborativeEditing, ExportWord, ExportPdf, Pagination, WProofreader, MathType' ],
+				toolbar: {
+					items: [
+						'undo', 'redo',
+						'|',
+						'exportPdf', 'exportWord',
+						'|',
+						'wproofreader', 'findAndReplace', 'selectAll',
```

Comment:
We should update this part same way as above.

Reply:
Addressed in commit.